packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -114,34 +143,49 @@ interface IMajorityVoting {
     ///@dev The function assumes the queried proposal exists.
     function canVote(uint256 _proposalId, address _voter) external view returns (bool);
 
-    /// @notice Method to execute a proposal if allowed to.
+    /// @notice Executes a proposal.
```

Comment:
```suggestion
    /// @notice Executes a proposal if approved.
```

Reply:
We would need to add "if approved/possible" also to the other methods that do checks, e.g., `createProposal`, `vote`, `changeVoteSettings`, etc.
Should we do this?

Reply:
If you think it makes things clear and it's reasonably fast, sure :+1: